front/migrations/20241211_parents_migrator.ts

```diff
@@ -143,7 +144,24 @@ const migrators: Record<ConnectorProvider, ProviderMigrator | null> = {
   google_drive: null,
   microsoft: null,
   github: null,
-  notion: null,
+  notion: {
+    transformer: (nodeId, parents) => {
```

Comment:
this one is not idempotent if it fails at some point you're in trouble right?

Reply:
it's quite idempotent

Reply:
Agreed